core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -556,6 +556,8 @@ private[spark] class Executor(
           }
         }
 
+        // update metric counting the number of tasks finished
+        executorSource.FINISHED_TASKS.inc(1L)
```

Comment:
Since you're making a point to expose this metric separately from the already existing "completeTasks", it probably should be named "succeededTasks" to emphasize the difference.

Reply:
I have changed this following the suggestion. Thanks.